kubernetes/images/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# clone the github repo
+git clone https://github.com/fossasia/susi_server.git /susi_server
+cd /susi_server
+git submodule update --recursive --remote
+git submodule update --init --recursive
+git checkout ${BRANCH}
+
+if [ -v COMMIT_HASH ]; then
+    git reset --hard ${COMMIT_HASH}
+fi
+
+rm -rf .git
```

Comment:
And I think, there would be a `.git` folder within `dependencies/public-transport-enabler` too. Since its a sub module.